content/docs/ref/runner.md

```diff
@@ -73,8 +73,191 @@ Any [generic option](/doc/ref) in addition to:
   (you'd need to write your code to save intermediate results to take advantage
   of this).
 
+
+### Using `--cloud-permission-set`
+
+The credentials must grant access to resources needed for managing compute instances.
+
+In general it is best to use the provider managed policies/roles, but to explicitly limit the credentials further is possible.
+
+<toggle>
+<tab title="AWS">
+
+- AWS Managed Policy: [`arn:aws:iam::aws:policy/AmazonEC2FullAccess`](https://us-east-1.console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/AmazonEC2FullAccess$serviceLevelSummary)
+
+**For example** this could potentially be further limited to:
+```
+ec2:CreateSecurityGroup -- (Firewall and SSH Access Management)
+ec2:AuthorizeSecurityGroupEgress
+ec2:AuthorizeSecurityGroupIngress
+ec2:DescribeSecurityGroups
+ec2:DescribeSubnets
+ec2:DescribeVpcs
+ec2:ImportKeyPair
+ec2:DeleteKeyPair
+ec2:CreateTags -- (General Resource Management)
+ec2:RunInstances -- (EC2 Instance Management)
+ec2:DescribeImages
+ec2:DescribeInstances
+ec2:TerminateInstances
+ec2:DescribeSpotInstanceRequests -- (Optionally needed for Spot Access)
+ec2:RequestSpotInstances
+ec2:CancelSpotInstanceRequests
+```
+
+</tab>
+<tab title="GCP">
+
+[GCP Managed Roles](https://cloud.google.com/iam/docs/understanding-roles#compute-engine-roles):
+- `roles/compute.admin`
+- `roles/iam.serviceAccountUse`
+
+**For example** this could potentially be further limited to:
+```
+compute.diskTypes.get
+compute.disks.create
+compute.firewalls.create
+compute.firewalls.delete
+compute.globalOperations.get
+compute.instances.create
+compute.instances.delete
+compute.instances.get
+compute.instances.list
+compute.instances.setMetadata
+compute.instances.setServiceAccount
+compute.instances.setTags
+compute.machineTypes.get
+compute.networks.create
+compute.networks.get
+compute.networks.updatePolicy
+compute.subnetworks.use
+compute.subnetworks.useExternalIp
+compute.zoneOperations.get
+compute.zones.get
+compute.zones.list
+iam.serviceAccounts.actAs
+```
+
+</tab>
+</toggle>
+
+You may also require additional permissions specific to your application (for example: object storage, private docker registries, and other cloud services).
+These additional permissions should be managed separately, and exposed either as
+independent credentials or via
+[`--cloud-permission-set`](https://cml.dev/doc/ref/runner#--cloud-permission-set)
+
+
 ## Examples
 
+### Using `--cloud-permission-set`
```

Comment:
`-Wduplicate-title`

Could we merge both `### Using --cloud-permission-set` into one big section? What's the reason to keep them separate?

Reply:
Sorry Using Both `### Using --cloud-permission-set` and ??? into one section?

Reply:
The heading `### Using --cloud-permission-set` currently appears twice in the file :)

Reply:
```suggestion

```